### DISCUSSION TITLE
wavelength range

### DISCUSSION THREAD

**Discussion by r0r0s** [BYUCamachoLab/simphony#81] [Q&A] [answered]
first of thanks for this software,

in olur research we use PICs as biosensors and because of the absorption spectrum
of water prefer wavelengths in the 850nm range.

Simphony prohibits simulations outside the 1500-1600nm range.
what is the underlying reason for this or should the models also work
for 850 if i manually remove the current wavelength restrictions?

thanks,
Rory Dijkink

**Comment by sequoiap** [comment, marked as answer]
Hi Rory,

If I remember my implementation correctly—Simphony checks the compact models for valid wavelength ranges when a simulation over a certain range is requested. The majority, if not all, of the compact models in Simphony are for the 1500-1600 nm range, but if you were to add your own compact models and s-parameters the simulation would then work as expected. But what's included in Simphony was not simulated for the 850 nm range. If you have any questions surrounding creating your own compact model library, I'd be happy to give some pointers.

Sequoia Ploeg
